internal/eventstore/v3/push2.sql

```diff
@@ -0,0 +1,78 @@
+WITH events AS (
+  select 
+    *
+    , TRANSACTION_TIMESTAMP() AS created_at
+    , EXTRACT(EPOCH FROM TRANSACTION_TIMESTAMP()) as position
+    , ROW_NUMBER() OVER(PARTITION BY owner, aggregate_type, aggregate_id) AS increment
+    , ROW_NUMBER() OVER() AS position_order
+  FROM (VALUES
+    %s
+  ) AS events (
+    instance_id
+    , "owner"
+    , aggregate_type
+    , aggregate_id
+    , revision
+
+    , creator
+    , event_type
+    , payload
+  )
+), aggregates AS (
+  SELECT
+    instance_id
+    , aggregate_type
+    , aggregate_id
+    , MAX("sequence") AS "sequence"
```

Comment:
I don't understand why this query wouldn't give a performance boost.
Is it possible that the `MAX` aggregate executes a scan on large aggregates, instead of using the primary key?